filebeat/docs/modules.asciidoc

```diff
@@ -9,4 +9,6 @@ modules.
 
 Filebeat modules require Elasticsearch 5.2 or later.
 
+NOTE: Bug fixes, enhancements, and new Filebeat modules are no longer implemented. We recommend that you {fleet-guide}/migrate-beats-to-agent.html[migrate to {agent}], whose integrations are actively maintained.
```

Comment:
```suggestion
NOTE: While {filebeat} modules are still supported, we recommend {agent} integrations over {filebeat} modules. Integrations provide a streamlined way to connect your data to the {stack}, and are available for popular services and platforms, such as {elastic-sec} and {observability}. Refer to the full list of integrations [here](https://www.elastic.co/integrations/data-integrations). 
```

Reply:
@jmikell821 can we reword 'available for popular services and platforms, such as {elastic-sec} and {observability}' as we're trying to convey that we cover data from popular services/platforms across a broad set of vendors/data sources. Such as Elastic sec/o11y doesn't make much sense in this case.

Reply:
@jamiehynds I edited the suggestion above ^ let me know if we need to make any additional tweaks, and how back this needs to be backported. Sorry for the initial misunderstanding.

Reply:
Looks good, thanks @jmikell821 - can we please ensure this text is also included? 

`Please refer to the [Beats vs Elastic Agent comparison](https://www.elastic.co/guide/en/fleet/current/beats-agent-comparison.html) documentation for more information.`